docs/usage/setup.md

```diff
@@ -153,14 +158,16 @@ Gateway settings:
 
   - **Just value**: The payload will contain only the row Numeric/String/Bool value
 
+- **Use nodes name instead of numeric nodeIDs**: When gateway type is `ValueId` use this flag to force to use node names instead of node ids in topic.
+- **Send Zwave Events**: Enable this to send all Zwave client events to MQTT. More info [here](#zwave-events)
+- **Include Node info**: Adds in ValueId json payload two extra values with the Name: `nodeName` and Location `nodeLocation` for better graphing capabilities (useful in tools like InfluxDb,Grafana)
 - **Ignore status updates**: Enable this to prevent gateway to send an MQTT message when a node changes its status (dead/sleep == false, alive == true)
 - **Ignore location**: Enable this to remove nodes location from topics
-- **Send Zwave Events**: Enable this to send all Zwave client events to MQTT. More info [here](#zwave-events)
-- **Include Node info**: Adds in ValueId json payload two extra values with the Name: `nodeName` and Location `nodeLocation` for better graphing capabilities (usefull in tools like InfluxDb,Grafana)
 - **Publish node details**: Creates an `nodeinfo` topic under each node's MQTT tree, with most node details. Helps build up discovery payloads.
-- **Use nodes name instead of numeric nodeIDs**: When gateway type is `ValueId` use this flag to force to use node names instead of node ids in topic.
```

Comment:
@towerhand This exists too, it's just hidden and appear when gateway type is 0

Reply:
The "use nodes name instead of numeric nodeId" is still there, I just moved it up a bit so it matches the order in the zwave2mqtt settings. I can move it back if you would like.

Reply:
Sorry missed that. It's ok :)